internal/page/donor/pay.go

```diff
@@ -0,0 +1,63 @@
+package donor
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/pay"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/sesh"
+)
+
+func Pay(
+	sessionStore SessionStore,
+	donorStore DonorStore,
+	payClient PayClient,
+	randomString func(int) string,
+	appPublicURL string,
+) Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request, donor *actor.DonorProvidedDetails) error {
```

Comment:
this has always been a weird thing on its own, when really it is just another handler

Reply:
Cool - finding this should be easier now too, I always forget its in register